collectors/ebpf.plugin/multi_metadata.yaml

```diff
@@ -705,7 +705,14 @@ modules:
     troubleshooting:
       problems:
         list: []
-    alerts: []
+    alerts:
+      - name: sync_freq
+        link: 
```

Comment:
@Ancairon  Can you please check this error?

Reply:
Some weird yamllint error, will try and see why it adds a newline there..